jbehave-core/src/test/java/org/jbehave/core/steps/ParameterConvertersBehaviour.java

```diff
@@ -1045,4 +1063,13 @@ public StringContainer convertValue(String value, Type type) {
             return new StringContainer(value);
         }
     }
+
+    private class ListToSetConverter extends AbstractChainableParameterConverter<List<String>, Set<String>> {
+
+        @Override
+        public Set<String> convertValue(List<String> value, Type type) {
+            return new HashSet<>(value);
+        }
+
```

Comment:
useless empty line

Reply:
do we have convention regarding to this in jbehave? I see many classes with empty before the last curly bracket